3ds/update_presence.go

```diff
@@ -1,13 +1,24 @@
 package friends_3ds
 
 import (
+	database_3ds "github.com/PretendoNetwork/friends-secure/database/3ds"
 	"github.com/PretendoNetwork/friends-secure/globals"
 	nex "github.com/PretendoNetwork/nex-go"
 	nexproto "github.com/PretendoNetwork/nex-protocols-go"
 )
 
-func UpdatePresence(err error, client *nex.Client, callID uint32, presence *nexproto.NintendoPresence, unknown bool) {
-	// TODO: Do something with this
+func UpdatePresence(err error, client *nex.Client, callID uint32, presence *nexproto.NintendoPresence, showGame bool) {
+	currentPresence := presence
+
+	// Send an entirely empty status, with every flag set to update
+	if !showGame {
+		currentPresence = nexproto.NewNintendoPresence()
+		currentPresence.GameKey = nexproto.NewGameKey()
+		currentPresence.ChangedFlags = 4294967295 // FF FF FF FF, All flags
```

Comment:
Large numbers like this should probbaly be represented as hex for easier ready. `0xFFFFFFFF` instead of 4294967295